Fix typo

zabio3 opened the pull request:
Misspell Finds commonly misspelled English words

 - gobgp/server/fsm.go
   - Line 1592: warning: "notificaiton" is a misspelling of "notification" (misspell)
 - gobgp/packet/bgp/bgp.go
   - Line 8432: warning: "Notificaiton" is a misspelling of "Notification" (misspell)
 - gobgp/config/bgp_configs.go
   - Line 2104: warning: "Configuraton" is a misspelling of "Configuration" (misspell)
   - Line 2136: warning: "Configuraton" is a misspelling of "Configuration" (misspell)
   - Line 2517: warning: "elasped" is a misspelling of "elapsed" (misspell)

fujita commented on internal/pkg/config/bgp_configs.go:
```diff
@@ -2212,7 +2212,7 @@ type RouteReflectorState struct {
 // struct for container bgp:config.
-// Configuraton parameters relating to route reflection
+// Configuration parameters relating to route reflection
 // for the BGP neighbor or group.
```
this file is automatically generated from OpenConfig yang files and can't be modified directly. The first two typos come from the original yang files so I guess that we had better to leave them alone.

The last one comes from:
https://github.com/osrg/gobgp/blob/master/tools/pyang_plugins/gobgp.yang

fujita commented:
Thanks! As I commented, you need to fix the yang file. If you are not familiar with how to generate go files from yang files, please drop the changes to bgp_configs.go. I'll take care of it.

zabio3 commented:
Thank you for commenting carefully.

I dropped the change of bgp_configs.go and changed the target part in gobgp.yang. (In that case, "Configuraton" is a misspelling of "Configuration" was not found, so only "elasped" is a misspelling of "elapsed" changed)

fujita commented:
thanks! I generated bgp_configs.go file from the yang file and squashed the commits.
